### DISCUSSION TITLE
只需要赞助6.66就可以使用解锁码了吗？

### DISCUSSION THREAD

**Discussion by Deep-Dark-Forest** [Meloong-Git/PCL#1843] [提问] [answered]
只需要赞助6.66元就可以使用解锁码了吗？
另外问一下6.66元是否包括5个基本主题？

**Comment by WForst-Breeze** [comment, marked as answer]
6.66 仅可以获得最新 PCL2 快照版本，>>**不提供后续快照版本更新**<<；可以使用活跃橙解锁码解锁活跃橙。

只要是快照版本就可以使用基本主题和隐藏主题。
